src/main/java/Duke.java

```diff
@@ -1,13 +1,39 @@
+import java.util.Scanner;
+
 public class Duke {
     /**
-     * Main method
+     * Main method.
      */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        System.out.printf("Hello I'm Duke\nWhat can I do for you\n");
+        while(true) {
+            Duke duke = new Duke();
+            String command = sc.nextLine();
+            if(duke.handleCommand(command)) {
+                break;   
+            }
+        }
+    }
+
+    /**
+     * Constructor.
+     */
+    public Duke() {}
+
+    /**
+     * Handles the various commands.
+     * @param command String
+     * @return boolean
+     */
+    boolean handleCommand(String command) {
```

Comment:
Very smart of you to come up with this method to return a suitable boolean! Well done!